### PR TITLE
Update wallet.md

### PR DESCRIPTION
**Pull request by Clay-Mysten**
Move SUI_ROOT creation from Wallet to Install

### CODE REVIEW

File: doc/src/build/wallet.md [hunk 1]
```diff
@@ -9,27 +9,28 @@ Sui wallet and execute wallet commands through its command line
 interface, *Wallet CLI*.
 
 
-## Setup
+## Set up
```
Comment (patrickkuo) [inline doc/src/build/wallet.md:12]: is this change intended? in `tutorials.md` we use Setup, we need to keep it consistent?

**Review by awelc — APPROVED** [review]

File: doc/src/build/wallet.md (hunk 1 above)
Comment (Clay-Mysten) [reply doc/src/build/wallet.md:12]: Yes and good catch. I've now fixed this in PR #1038 1038. We are using "setup" like a verb, but that's a noun. "Set up" is a verb.
